### PR TITLE
Merge commits for release 161.8834.0

### PR DESCRIPTION
**Pull request by llali**
Merge commits for release 161.8834.0

### CODE REVIEW

**Review by barbaravaldez — CHANGES_REQUESTED** [review]
Can you revert the commit [4dd0fe8](https://github.com/microsoft/SqlScriptDOM/pull/44/commits/4dd0fe8ed74185335e08a1c2d80b8c31ebc04d2e)? This change was originally made in Github so no need to add it again

**Comment by llali** [comment]
> Can you revert the commit [4dd0fe8](https://github.com/microsoft/SqlScriptDOM/pull/44/commits/4dd0fe8ed74185335e08a1c2d80b8c31ebc04d2e)? This change was originally made in Github so no need to add it again

done

**Review by zijchen — APPROVED** [review]

File: Directory.Build.props [hunk 1]
```diff
@@ -49,11 +49,28 @@
...
+   -->
+  <PropertyGroup>
+    <EnableLocalization Condition="'$(EnableLocalization)' == ''">false</EnableLocalization>
+    <PostBuildLocalization>$(EnableLocalization)</PostBuildLocalization>
+    <!-- This corresponds to the APEX13 languages: Tier1 + CSY/TRK/PLK
+         For language sets details, see: https://ceapex.visualstudio.com/CEINTL/_git/SoftwareLocalization?path=%2Fsrc%2FOneBranchPackages%2FLocalization.Languages%2FLocalization.Languages.props&_a=contents&version=GBmain
+    -->
+    <LanguageSet Condition="'$(EnableLocalization)' == 'true'">VS_Main_Languages</LanguageSet>
+  </PropertyGroup>
+
```
Comment (barbaravaldez) [inline Directory.Build.props:73]: We can either remove these properties or keep them. Since they are not needed in the github repo

**Review by barbaravaldez — APPROVED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (llali) [reply Directory.Build.props:73]: that's fine we can keep it to make future merge easier

**Review by barbaravaldez — APPROVED** [review]

**Pull request merged**